src/utils/getPlatformString.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { BuildImageCommandOptions } from "../runtimes/docker/contracts/ContainerClient";
+
+/**
+ * This method parses the `platform` field in tasks.json
+ * @returns Platform string specified by `platform`
+ */
+export function getPlatformString(options: BuildImageCommandOptions): string {
```

Comment:
This helper should probably live in the DockerClientBase folder, since it's a calculating a Docker/Go specific value for the client.